modules/react/checkbox/lib/Checkbox.tsx

```diff
@@ -322,6 +323,13 @@ export const Checkbox = createComponent('input')({
     Element
   ) => {
     const inputId = useUniqueId(id);
+    const {localRef, elementRef} = useLocalRef(ref);
+    React.useEffect(() => {
+      if (typeof indeterminate === 'boolean' && localRef.current) {
+        localRef.current.indeterminate = indeterminate;
```

Comment:
@vibdev Last bit: can you add a Cypress test ensuring `indeterminate` is being set properly on the checkbox?

I'm thinking something which loads the Indeterminate story, checks a topping (any of them), and then verifies the `Supreme Pizza Toppings` checkbox has the `indeterminate` property set. Something like this should do the trick:

```
cy.findByLabelText('Supreme Pizza Toppings').should('have.prop', 'indeterminate', true);
```